test/test_helper.rb

```diff
@@ -39,6 +39,15 @@ def psych_load(*args)
 
 # Live test for our implementation of Ruby's compaction methods (rb_gc_mark_movable
 # and rb_gc_location), enabling compaction for every major collection.
-if defined?(GC.auto_compact) == "method"
-  GC.auto_compact = true
+GC.auto_compact = true if defined?(GC.auto_compact) == "method"
+
+# Formats the WKT heredoc string into something that will match the
+# {RGeo::WKRep::WKTGenerator#generate} method.
+#
+# @param wkt [String]
+#
+# @return [string]
+#
+def format_wkt(wkt)
```

Comment:
could we avoid that by feeding wkt that is accepted? I think that layer of direction doesn't make tests clearer :/

Reply:
I won't be able to use heredoc then as the format that is generated by the code won't match the format from the heredoc. I'll remove all heredoc and just use strings

Reply:
Yeah I think strings are fine